### PR TITLE
Replace of 11ty template. Header, footer.

### PR DESCRIPTION
**Pull request by error404as**

### CODE REVIEW

File: src/assets/css/_fonts.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@font-face {
+    font-family: 'Helvetica Neue';
+    src: local('Helvetica Neue'),
+        url('../../fonts/HelveticaNeueCyr-Medium.woff2') format('woff2'),
```
Comment (alexeykomov) [inline src/assets/css/_fonts.scss:4]: Let's not carry external fonts and rely only on system ones.

File: .browserslistrc [hunk 2]
```diff
@@ -0,0 +1,2 @@
+> 1%
+last 2 versions
```
Comment (alexeykomov) [inline .browserslistrc:3]: 👍 
I suggest to add **iOS 12+ mobile Safari** rule, since iOS app works on iOS 12+.

File: src/assets/css/main.scss [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+.btn-primary {
+    display: inline-flex;
+    vertical-align: middle;
+    align-items: center;
+    height: 48px;
+    padding: 0 40px;
+    border: 0;
+    border-radius: 4px;
+    background: $color-btn-bg;
+    font: 14px/1 Helvetica, sans-serif;
```
Comment (alexeykomov) [inline src/assets/css/main.scss]: Let's rely on system fonts.

File: src/assets/css/main.scss [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+}
+
+.u-visually-hidden {
+    position: absolute !important;
+    width: 1px !important;
+    height: 1px !important;
+    clip: rect(0 0 0 0);
+    padding:0 !important;
+    border:0 !important;
+    overflow: hidden;
```
Comment (alexeykomov) [inline src/assets/css/main.scss:25]: I saw such [visually hidden class](https://www.scottohara.me/blog/2017/04/14/inclusively-hidden.html).
```suggestion
.visually-hidden {
  clip: rect(0 0 0 0); 
  clip-path: inset(50%);
  height: 1px;
  overflow: hidden;
  position: absolute;
  white-space: nowrap; 
  width: 1px;
}
```

File: src/assets/js/google.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
+document.addEventListener('DOMContentLoaded', () => {
+    if (window.location.hostname === 'localhost') {
+        return;
+    }
+    const id = 'YOUR_ID_HERE';
```
Comment (alexeykomov) [inline src/assets/js/google.js:5]: I assume this will be changed at build time.

File: src/_components/lang.ejs [hunk 6]
```diff
@@ -0,0 +1,6 @@
+<div class="c-lang">
```
Comment (alexeykomov) [inline src/_components/lang.ejs:4]: Screenshot 2022-07-08 at 15 17 19
Isn't it a &lt;select&gt; element?

File: src/_data/cacheBust.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
+const md5File = require('md5-file');
```
Comment (alexeykomov) [inline src/_data/cacheBust.js:1]: Is this included into production bundle?

File: src/assets/css/_fonts.scss (hunk 1 above)
Comment (alexeykomov) [reply src/assets/css/_fonts.scss:4]:
```
/* System Fonts as used by GitHub */
body {
  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
}
```

Comment (error404as) [reply src/assets/css/_fonts.scss:4]: Ok, as you wish :)

File: .browserslistrc (hunk 2 above)
Comment (error404as) [reply .browserslistrc:3]: Done

File: src/assets/css/main.scss (hunk 3 above)
Comment (error404as) [reply src/assets/css/main.scss]: Removed font family

File: src/assets/css/main.scss (hunk 4 above)
Comment (error404as) [reply src/assets/css/main.scss:25]: Updated

File: src/assets/js/google.js (hunk 5 above)
Comment (error404as) [reply src/assets/js/google.js:5]: Currently file is not used

File: src/_data/cacheBust.js (hunk 7 above)
Comment (error404as) [reply src/_data/cacheBust.js:1]: Used only while build for file names

**Review by alexeykomov — APPROVED** [review]

**Pull request merged**